PWGCF/Flow/Tasks/FlowZDCtask.cxx

```diff
@@ -313,8 +327,38 @@ struct FlowZDCtask {
       }
     }
   }
+  void processNeutronSkin(aodCollisions::iterator const& collision, aod::Zdcs const& zdcs, aod::FT0s const& ft0)
+  {
+    float cent = collision.centFT0C();
+    auto ft0Entry = ft0.iteratorAt(collision.globalIndex());
+    float sumA = ft0Entry.sumAmpA();
+    float sumC = ft0Entry.sumAmpC();
+    float FT0MAmp = sumA + sumC;
+    histos.fill(HIST("hFT0MAmp"), FT0MAmp);
+
+    // Access ZDC data
+    auto zdcEntry = zdcs.iteratorAt(collision.globalIndex());
```

Comment:
In what I know the `FT0` table and the `ZDC` are not addressed by collision index
Are you sure you are doing what you want to?